lib/rails_erd/diagram/graphviz.rb

```diff
@@ -188,6 +188,9 @@ def relationship_style(relationship)
         # Title of the graph itself.
         graph[:label] = "#{title}\\n\\n" if title
 
+        # Style of splines
+        graph[:splines] = options.splines
```

Comment:
Is there a default setting for this? If `option.splines` returns a `nil`, what happens?